src/evmTest/scala/io/iohk/ethereum/vm/ThrowSpec.scala

```diff
@@ -13,7 +13,7 @@ class ThrowSpec extends FreeSpec with Matchers {
 
       val result = contract.justThrow().call()
 
-      result.error shouldBe Some(InvalidOpCode) // TODO: change to invalid jump destination
+      result.error shouldBe Some(InvalidOpCode(0xfd.toByte))
```

Comment:
I just installed a new version and I have the same thing. It seems we have 2 options:

* require a specific solc version - fine for CI, not so much for developers
* bit a bullet and deal with those test becoming broken - hopefully that's not going to happen very often